google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -124,6 +125,48 @@ public ManagedChannelBuilder apply(ManagedChannelBuilder input) {
     return builder;
   }
 
+  /**
+   * Enables Opencensus metric aggregations.
```

Comment:
minor - change all occurrences of "Opencensus" to "OpenCensus"

Reply:
done